app/models/user.rb

```diff
@@ -0,0 +1,3 @@
+class User < ActiveRecord::Base
+  # TODO: カラム追加
```

Comment:
__[rubocop]__
* Style/AsciiComments: Use only ascii symbols in comments. [:ref](https://github.com/bbatsov/ruby-style-guide#english-comments)